jenkins/build.sh

```diff
@@ -68,19 +123,25 @@ INSTALL_PATH="/opt/seagate/""${PRODUCT}"
 
 mkdir -p "$INSTALL_PATH"
 
+# build addb plugin
+if [ "$BUILD_ADDB" == "true" ]; then
+  echo "building addb plugin and adding addb binary file (rgw_addb_plugin.so)\
```

Comment:
echo "Building addb plugin(rgw_addb_plugin.so) for RGW and bundling same \
  in cortx-rgw-integration rpm."

Reply:
done